src/fhir/CarePlan.DCW.CogLevel1.NoDementiaDx.json

```diff
@@ -0,0 +1,75 @@
+{
+    "resourceType": "CarePlan",
+    "id": "TBD",
+    "description": "DCW cognition domain, level 1 (which is only for patients who don't have a dementia Dx)",
+    "status": "active",
+    "intent": "order",
+    "category": [
+        {
+            "coding": [
+                {
+                    "system": "http://snomed.info/sct",
+                    "code": "719091000000102",
+                    "display": "Questionnaire"
+                }
+            ],
+            "text": "Questionnaire"
+        }
+    ],
+    "subject": {
+        "reference": "Patient/TBD",
+        "display": "Doe, John TBD"
+    },
+    "activity": [
```

Comment:
@mcjustin  does order of these matter?

Reply:
@achen2401  Let's try making the activity array order indicate the sequence in which to present questionnaires. I'm slightly concerned that this order won't be preserved by HAPI consistently, but if we find that's the case we can change course. I think the more FHIR-compliant route here would be for the CarePlan to use https://www.hl7.org/fhir/requestgroup.html , but let's save that for a later phase.